app/src/main/java/com/example/fivenightsescape/MapsActivity.kt

```diff
@@ -28,13 +32,33 @@ class MapsActivity : AppCompatActivity(), OnMapReadyCallback {
     private lateinit var fusedLocationClient: FusedLocationProviderClient
     private lateinit var mMap: GoogleMap
 
+    //timer and lv things
+    private var levelChanger = 4 //levelChanger * delay = every how many milliseconds next lv, it changes every two levels
+    private val delay: Long = 500 // how often does the progressbar update
+    private var currentLevel = 1 //lv the player is on
+    private var subLevel = 0 // for the progressbar and to change level length
+    private val handlerTimer = Handler()
+    private lateinit var levelText: TextView
+    private lateinit var progressBar: ProgressBar
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_maps)
 
+        //timer
+        val timerONE :Chronometer = findViewById(R.id.timerONE)
+        timerONE.start()
+        handlerTimer.postDelayed(newLevel, delay)
+        //level indicator
+        levelText = findViewById(R.id.levelIndicator)
```

Comment:
%2
Add here:
```kt
levelText.text = getString(R.string.level, currentLevel)
```
So that when user changes activity it will change the string from: `Level: %1$d` to `Level: 1`